pkgs/tools/misc/ipxe/default.nix

```diff
@@ -27,10 +30,12 @@ stdenv.mkDerivation {
     [ "ECHO_E_BIN_ECHO=echo" "ECHO_E_BIN_ECHO_E=echo" # No /bin/echo here.
       "ISOLINUX_BIN_LIST=${syslinux}/share/syslinux/isolinux.bin"
       "LDLINUX_C32=${syslinux}/share/syslinux/ldlinux.c32"
-    ] ++ lib.optional (embedScript != null) "EMBED=${embedScript}";
+    ] ++ lib.optional (trustedRootCert != null) "TRUST=${trustedRootCert}"
+      ++ lib.optional (embedScript != null) "EMBED=${embedScript}";
 
 
-  enabledOptions = [ "DOWNLOAD_PROTO_HTTPS" ];
+  enabledOptions = lib.optional optionDownloadProtoHttps "DOWNLOAD_PROTO_HTTPS"
```

Comment:
Hmm why add an option for it?

Reply:
I thought rather than arbitrarily hard-coding certain options while exposing others, it might be better to allow each supported option to be set. I defaulted this option to the previous value (enabled).